com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -59,7 +59,7 @@ static class Styles
                 new GUIContent("Both"),
             };
             public static int[] xrTargetEyeValues = { 0, 1 };
-            public static readonly GUIContent xrTargetEye = EditorGUIUtility.TrTextContent("Target Eye", "Allows XR rendering if target eye sets to both eye. Disable XR for this camera otherwise.");
+            public static readonly GUIContent xrTargetEye = EditorGUIUtility.TrTextContent("Target Eye", "Allows XR rendering if target eye sets to both eye. Disable XR for this camera otherwise. When XR is disabled, MSAA UI will be hide. All XR cameras share the same MSAA level sepcified in URP PipelineAsset Settings. All effects that are not compatible with XR will be disabled as well, such as lens distortion.");
```

Comment:
The doc string that has been approved for trunk looks like this.
https://ono.unity3d.com/unity/unity/pull-request/112453/_/graphics/camera-editor-tooltip-fix
I don't know if we need more than that. But I will let @oleks-k decide that. He can also convey with Chris Butta on this.

Reply:
@keiramoon Maybe you have some input on this one?

Reply:
Change "if target eye sets to both eye" to "if the target eye is set to both eyes". This is an instance where passive voice is okay (the target doesn't set itself, something else sets it).

Reply:
The tooltip sounds confusing to me, but that could be because I'm not so familiar with the XR functionality.
My suggestion:
"Select which eye of an XR device the Camera renders to. Selecting the option Both enables XR rendering on this Camera, selecting other options disables the XR rendering. Enabling XR rendering on a Camera:\n* Disables the MSAA property in the Camera Inspector. The URP asset defines the MSAA level for all XR Cameras.\n* Disables all effects that are not compatible with XR rendering, for example, lens distortion."

Reply:
Thanks guys, I have changed the tooltip to follow @oleks-k 's suggestion at b62fc3a :) 
Much better!

Reply:
@keiramoon I have updated the tooltips at b62fc3a, Could you review it again? Thanks